AI/tests/check_positions.py

```
@@ -0,0 +1,148 @@
+
+def plot_ticker_trades(ticker: str, conn):
+    """특정 종목의 주가 차트와 매매 타점을 그립니다."""
+    try:
+        # 주가 데이터 로드
+        price_query = f"""
+            SELECT date, close 
+            FROM public.price_data 
+            WHERE ticker = '{ticker}' 
+              AND date >= '2025-08-01'
+            ORDER BY date ASC;
+        """
```

Ruff (0.15.2)

[error] Possible SQL injection vector through string-based query construction

(S608)